Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.h

```diff
@@ -83,6 +83,7 @@ WARN_UNUSED_RETURN GstPad* webkitGstGhostPadFromStaticTemplate(GstStaticPadTempl
 bool getVideoSizeAndFormatFromCaps(const GstCaps*, WebCore::IntSize&, GstVideoFormat&, int& pixelAspectRatioNumerator, int& pixelAspectRatioDenominator, int& stride, double& frameRate, PlatformVideoColorSpace&);
 std::optional<FloatSize> getVideoResolutionFromCaps(const GstCaps*);
 bool getSampleVideoInfo(GstSample*, GstVideoInfo&);
+bool getDisplaySize(WebCore::IntSize originalSize, int pixelAspectRatioNumerator, int pixelAspectRatioDenominator, guint& width, guint& height);
```

Comment:
So we are returning another size, meaning width and height. We should do it with another IntSize object instead of output parameters. As we need to know if the conversion was successful, we should wrap it inside a std::optional. Something like:

```
std::optional<WebCore::IntSize> computeDisplaySize(WebCore::IntSize, int pixelAspectRatioNumerator, int pixelAspectRatioDenominator);
```